packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/aspnet/enrollmentintegrationsteps.md

```diff
@@ -1 +1,328 @@
+    }).then((newCredential) => {
+        const clientDataJSON = binToStr(
+            newCredential.response.clientDataJSON);
+        const attestationObject = binToStr(
+            newCredential.response.attestationObject);
+
+        const params = {
+            "clientData": clientDataJSON,
+            "attestation": attestationObject,
+            "challenge": challenge,
+            "userId": userId,
+            "username": username,
+            "displayName": displayName
+        };
+
+        const options = {
+            method: 'POST',
+            body: JSON.stringify(params),
+            headers: { "Content-type": "application/json; charset=UTF-8" }
+        };
+```
+
+### 7. Send verification credentials to Okta server
+
+Send the verifying credentials back to the Okta server to finish enrolling the WebAuthn authenticator for the user. Okta will decrypt the challenge using the user's public key and store the user's credentials and public key.
+
+```js
+        fetch("@Url.Action("EnrollWebAuthnAuthenticatorAsync", "Manage")", options)
+            .then(res => {
+                console.log("Request successful! Response:", res);
+                location.href = '@Url.Action("EnrollWebAuthnAuthenticator", "Manage",
+                    new { verificationCompleted = true})';
+            }).catch(function(err) {
+                console.error(err);
+                }
+            );
+    }).catch(function (err) {
+        console.error(err);
+    });
+}, false);
+```
+
+Pass the credentials as a parameter to the `EnrollAuthenticatorAsync` method on the `IdxClient`.
+
+```csharp
+var authnResponse = await _idxClient.EnrollAuthenticatorAsync(
+    new EnrollWebAuthnAuthenticatorOptions
+    {
+        Attestation = viewModel.Attestation,
+        ClientData = viewModel.ClientData,
+    }, (IIdxContext)Session["idxContext"]);
+
+Session["WebAuthnResponse"] = authnResponse;
+```
+
+### 8: Enroll More Authenticators or Log the User In
+
+Query the `AuthenticationStatus` property of the `AuthenticationResponse` object returned by `EnrollAuthenticatorAsync` to discover the current status of the authentication process. You'll need to respond to two specific authenticator statuses:
+
+* `AwaitingAuthenticatorEnrollment`
+* `Success`
+
+A status of `AwaitingAuthenticatorEnrollment` means that there are other authenticator types (Google, Okta Verify) which can be enrolled by the user and that your app should return to Step 4 and display a list of those authenticators which are still unenrolled as well as an option to skip further enrollment.
+
+A status of `Success` (or the user choosing to skip further authenticator enrollment) means that the user has now successfully enrolled their WebAuthn authenticator and is logged into the app. Call `AuthenticationHelper.GetIdentityFromTokenResponseAsync` to retrieve the OIDC claims information about the user and pass them into your application.
```

Comment:
```suggestion
A status of `Success` (or the user choosing to skip further authenticator enrollment) means that the user is now successfully enrolled using the WebAuthn authenticator and is signed in to the app. Call `AuthenticationHelper.GetIdentityFromTokenResponseAsync` to retrieve the OIDC claims information about the user and pass them into your application.
```

Reply:
@susanharper-okta  This change implies something slightly different from what I understand is happening.